com.unity.formats.alembic/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 ## [2.2.0-exp.3] - 2021-01-22
 ### Changes
 - Minimum Unity version is 2019.4
+- Added a custom dependency on the "CurrentRenderPipeline" setting and assigning the correct SRP by default.
```

Comment:
Suggestion:
`- Added a custom dependency on the "CurrentRenderPipeline" setting and ensured to assign the correct SRP by default.`